src/Thycotic.SecretServer/Thycotic.SecretServer.csproj

```diff
@@ -15,7 +15,6 @@
     <PackageReference Include="PowerShellStandard.Library" Version="7.0.0-preview.1">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="RestSharp" Version="106.11.8-alpha.0.14"/>
```

Comment:
Can this also be updated to the current stable release?

Reply:
Update the RestSharp package to 106.15.0